plugins/che-plugin-bayesian-lang-server/src/main/java/org/eclipse/che/plugin/languageserver/bayesian/server/launcher/BayesianLanguageServerLauncher.java

```diff
@@ -64,19 +72,21 @@ protected LanguageServer connectToLanguageServer(
   }
 
   protected Process startLanguageServerProcess(String projectPath) throws LanguageServerException {
-    String launchCommand = launchScript.toString();
+    String recommenderToken = null;
 
-    if (keycloakStore.hasLastToken()) {
-      launchCommand =
-          "export RECOMMENDER_API_TOKEN="
-              + //
-              keycloakStore.getLastTokenWithoutBearer()
-              + //
-              " && "
-              + //
-              launchCommand;
+    try {
+      String endpoint = apiEndpoint + "/bayesian/token";
+      LOG.info("Retrieving the Bayesian recommender token from API : {}", endpoint);
+      recommenderToken = httpJsonFactory.fromUrl(endpoint).request().asString();
+    } catch (Exception e) {
+      throw new LanguageServerException("Can't start Bayesian language server", e);
     }
 
+    LOG.info("Recommender token : {}", recommenderToken);
```

Comment:
is that token should be displayed by default ?

Reply:
No, you're right, it's a mistake.

Reply:
no, I should have removed this after tests